src/components/Match/matchPages.jsx

```diff
@@ -232,9 +232,12 @@ const matchPages = [{
 }, {
   name: strings.tab_cosmetics,
   parsed: true,
-  content: match => (<div>
-    <TeamTable match={match} columns={cosmeticsColumns} heading={strings.heading_cosmetics} />
-  </div>),
+  content: match => (
+    <div>
+      <Heading title={strings.heading_cosmetics} />
```

Comment:
shouldn't we be consistent and use split tables everywhere if we are going to use them at all?  Seems strange to break from that just for cosmetics.